index.html

```diff
@@ -19,268 +19,375 @@
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <i class="fa-brands fa-linkedin-in"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-angellist"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-twitter"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-medium"></i>
-              </a>
-            </li>
-          </ul>
-        </li>
-        <li>
-          <div id="button-container">
-            <button type="button" class="btn">
-              <a href="#">Get my resume</a>
-            </button>
+      <article class="flex" id="about">
+        <div id="description-container">
+          <div class="flex" id="description">
+            <h2 class="header">
+              <span>About<br />Myself</span>
+            </h2>
+            <p class="text">
+              I’m a software developer! I can help you build a product, feature
+              or website. Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h2 id="connect">Let's Connect</h2>
+            <div id="social-icons-container">
+              <ul id="social-icons">
+                <li class="icon">
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-github"></i>
+                  </a>
+                </li>
+                <li class="icon">
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-linkedin-in"></i>
+                  </a>
+                </li>
+                <li class="icon">
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-angellist"></i>
+                  </a>
+                </li>
+                <li class="icon">
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-twitter"></i>
+                  </a>
+                </li>
+                <li class="icon">
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <img src="./images/medium.png" alt="medium" />
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn">
+                <a href="#">Get my resume</a>
+              </button>
+            </div>
```

Comment:
> - [ ] As the project requires you to replicate the design as much as possible. It's important to stick with the Figma design because it serves as a blueprint or guide for the look and feel of your website. Deviating from the design may lead to inconsistencies or confusion for users, which could negatively impact their experience. Please kindly ensure to stick with the Figma design.😊
> 
> Figma design👇:
> 
> ![about1-figma](https://user-images.githubusercontent.com/40009354/236545451-83d4621b-306b-4e7d-9864-397da46170f4.png)
> 
> Your design👇:
> 
> ![about1](https://user-images.githubusercontent.com/40009354/236545733-d87dd77b-2f5b-44b5-875c-d3e6d80ca710.png)
>